BitwardenShared/Core/Platform/Models/Domain/Account.swift

```diff
@@ -132,7 +132,7 @@ extension Account {
         // MARK: Properties
 
         /// The environment URLs for an account.
-        var environmentUrls: EnvironmentUrlData?
+        var environmentURLs: EnvironmentURLData?
```

Comment:
❓ I think this is persisted to UserDefaults, is this going to cause decoding existing values to fail?

Reply:
Good question. I'll double-check that to make sure.

Reply:
Having checked, it does cause decoding existing values to fail. Given a choice between either manually specifying a coding key (to maintain backwards compatibility), creating a migration, or just reverting the case change on the variable; I think the last makes the most sense. A coding key just feels weird, and a migration is too heavyweight, and the casing of these variables isn't so crucial that we can't just live with it being cased like it was.